docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,147 @@
+# Migrate from other CNI to Antrea
```

Comment:
The whole file is not using a consistent line width.

Reply:
Reformated this doc with a consistent line width, now all lines are less than 80 characters.